comp/core/tagger/common/constants.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+// Package common provides common constants and methods for the tagger component and implementation
+package common
+
+import "github.com/DataDog/datadog-agent/comp/core/tagger/types"
+
+const (
```

Comment:
I think we should check in the code the usages of this hardcoded constants and replace them with the ones here. There are some usages of .Tag that hardcode the prefix for example.